popup/popup.html

```diff
@@ -9,15 +9,34 @@
       There is no live stream currently running.
     </div>
     <div id="check_options" class="popup-font popup-info">
-      To get live stream notifications, enter your API key and turn on the Live
-      Stream Notifications option in the Add-On Options page.
+      To use the live stream indicator and quick nav, enter your API key below:
+      <div class="option-text">
+        <div class="option-infobutton-container">
+          <img src="../options/infobutton.png"/>
+          <div class="option-infobutton-text">
+            This extension uses the Giant Bomb API. This API requires
+            the use of an API key, and the extension has a default key. However, the
+            API has usage rate limits per API key, so it is recommended that you provide
+            your own for better reliability. You can find your key by logging in to
+            Giant Bomb and navigating to <a href="http://www.giantbomb.com/api" target="_blank">giantbomb.com/api</a>.
+            Your key will never be logged or shared by Giant Bomb QoL.
+          </div>
+        </div>
+      </div>
+      <div class="centered">
+        <input type="text" id="text_api_key" placeholder="API Key"/><br>
+      </div>
+    </div>
+    <div id="notifications_disabled" class="popup-font popup-info">
+      Stream notifications are currently disabled.
     </div>
     <div id="live_stream_info" class="popup-buttons">
       <img id="stream_image" class="stream-link">
       <h2 id="stream_title" class="popup-font stream-link"></h2>
     </div>
     <div id="popup_bottom_row">
       <div id="options_link" class="popup-buttons popup-font" href="#">Add-On<br>Options</div>
+      <div id="stream_notifications" class="popup-buttons popup-font" href="#">Stream<br>Notifications</div>
```

Comment:
In that case, to unify this design, we should also remove the toggle from the extension options.  The only reason this component is present is to prevent the user from ever having to see the extension options panel for UX reasons.

Reply:
That's fair.  Should I save the code for it in case we decide to do something like it with the rest of the options?

Reply:
Yeah, that'd probably be a wise idea. If you know of an elegant way of preserving it so it's easy to find in the future that'd be great, otherwise just commenting out and putting a TODO is fine.